windows/ReactTestApp/ReactInstance.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+namespace winrt::ReactTestApp::implementation
+{
+    enum class JSBundleSource {
+        DevServer,
+        Embedded,
+    };
+
+    class ReactInstance
+    {
+    public:
+        winrt::Microsoft::ReactNative::ReactNativeHost ReactHost();
+        void LoadJSBundleFrom(JSBundleSource source);
+
+    private:
+        winrt::Microsoft::ReactNative::ReactNativeHost m_reactNativeHost;
```

Comment:
Nit: For class members, let's postfix `_` instead of prefixing `m_`:
```suggestion
        winrt::Microsoft::ReactNative::ReactNativeHost reactNativeHost_;
```